Tests/LibWeb/AccessibilityBridge/tests/test_tree_shape.py

```
@@ -0,0 +1,140 @@
+                    p, 0, Atspi.Text.get_character_count(p) or 0
+                ):
+                    parent = p.get_parent()
+                    # The immediate parent of this paragraph should be the document, *not* an unnamed generic/section.
+                    self.assertEqual(
+                        parent.get_role_name(),
+                        "document web",
+                        f"parent role was {parent.get_role_name()!r}, expected document web",
+                    )
+                    return
+            except Exception:
+                continue
```

Ruff (0.16.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)